packages/@n8n/nodes-langchain/utils/helpers.ts

```diff
@@ -64,3 +65,15 @@ export function getPromptInputByType(options: {
 
 	return input;
 }
+
+export function logAiEvent(
+	executeFunctions: IExecuteFunctions,
+	event: EventNamesAiNodesType,
+	data?: IDataObject,
+) {
+	try {
+		void executeFunctions.logAiEvent(event, data ? jsonStringify(data) : undefined);
```

Comment:
Since we're not `await`-ing here, I think the `try...catch` won't catch it, would it? We should make this function async and await for the `logAiEvent` call at least wdyt?

Reply:
You're right, it wouldn't, changed it to async/await. Thanks!